src/App/Fossa/Config/Common.hs

```diff
@@ -62,10 +62,10 @@ import Control.Effect.Diagnostics (
   (<||>),
  )
 import Control.Effect.Lift (Lift, sendIO)
-import Control.Timeout (Duration (Minutes))
+import Control.Timeout.Internal ( Duration(Minutes) )
```

Comment:
Don't import this from `Internal`.  I really want to prevent VSCode from suggesting that, but for now we have to live with it.

Reply:
Can we `hlint` against `*.Internal` modules?

Reply:
No, because we have to sometimes import internal modules.  There are certain ways of fully re-exporting symbols that don't get refined like this, (e.g. `Has` gets imported from effect modules, but is really from `Control.Algebra`), we just need to use those for the `Control.Timeout` module.

Reply:
Corrected on d68974e